sdk/tables/aztables/README.md

```diff
@@ -0,0 +1,174 @@
+- `TableClient` - Client that provides methods to interact at an table entity level such as creating, querying, and deleting entities within a table.
+- `Table` - Tables store data as collections of entities.
+- `Entity` - Entities are similar to rows. An entity has a primary key and a set of properties. A property is a name value pair, similar to a column.
+
+Common uses of the Table service include:
+
+- Storing TBs of structured data capable of serving web scale applications
+- Storing datasets that don't require complex joins, foreign keys, or stored procedures and can be de-normalized for fast access
+- Quickly querying data using a clustered index
+- Accessing data using the OData protocol and LINQ filter expressions
+
+### Thread safety
+We guarantee that all client instance methods are thread-safe and independent of each other ([guideline](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-service-methods-thread-safety)). This ensures that the recommendation of reusing client instances is always safe, even across threads.
+
+### Additional concepts
+<!-- CLIENT COMMON BAR -->
+[Client options]() |
+[Accessing the response]() |
+[Long-running operations]() |
+[Handling failures]() |
+[Diagnostics]() |
+[Mocking]() |
+[Client lifetime](https://devblogs.microsoft.com/azure-sdk/lifetime-management-and-thread-safety-guarantees-of-azure-sdk-net-clients/)
+<!-- CLIENT COMMON BAR -->
+
+## Examples
+- [Create the Table service client](#create-the-table-service-client)
+    - [Create an Azure table](#create-an-azure-table)
+    - [Get an Azure table](#get-an-azure-table)
+    - [Delete an Azure table](#delete-an-azure-table)
+- [Create the Table client](#create-the-table-client)
+    - [Add table entities](#add-table-entities)
+    - [Query table entities ](#query-table-entities)
+    - [Delete table entities](#delete-table-entities)
+
+### Create the Table service client
+
+First, we need to construct a `TableServiceClient`.
+
+
+
+### Create an Azure table
+Next, we can create a new table.
+
+
+### Get an Azure table
+The set of existing Azure tables can be queries using an OData filter.
+
+
+### Delete an Azure table
+
+Individual tables can be deleted from the service.
+
+
+### Create the Table client
+
+To interact with table entities, we must first construct a `TableClient`.
+
+
+### Add table entities
+
+Let's define a new `TableEntity` so that we can add it to the table.
+
+Using the `TableClient` we can now add our new entity to the table.
+
+
+### Query table entities
+
+To inspect the set of existing table entities, we can query the table using an OData filter.
+
+
+If you prefer LINQ style query expressions, we can query the table using that syntax as well.
+
+
+### Delete table entities
+
+If we no longer need our new table entity, it can be deleted.
+
+
+## Troubleshooting
+
+When you interact with the Azure table library using the .NET SDK, errors returned by the service correspond to the same HTTP 
```

Comment:
I think any reference to .NET should be removed before merging. We can add the Go specific information later.